### PR TITLE
remove pandas

### PR DESCRIPTION
**Pull request by mayabrandi**
### Review:
- [ ] Code approved by
- [x] Tests executed by @mayabrandi 
- [ ] "Merge and deploy" approved by

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by mayabrandi** [comment]
tested locally agains step 24-174163 
Skärmavbild 2021-02-16 kl  17 08 30

File: cg_lims/EPPs/files/file_to_udf.py [hunk 1]
```diff
@@ -41,18 +41,18 @@ def set_udfs(well_field: str, value_field: str, udf: str, well_dict: dict, resul
...
-        art = well_dict[well]
-        art.udf[udf] = value
-        art.put()
-        passed_arts += 1
+    with open(result_file, newline="", encoding="latin1") as csvfile:
+        reader = csv.DictReader(csvfile)
+        for sample in reader:
+            well = sample.get(well_field)
+            value = sample.get(value_field)
+            if not value or math.isnan(value) or well not in well_dict:
```
Comment (moonso) [inline cg_lims/EPPs/files/file_to_udf.py:49]: Did you test this? Not sure if `math.isnan` is applicable here since this is not pandas anymore...

**Review by moonso — APPROVED** [review]
Looks good! I only had one comment. And don't forget to remove pandas from the requirements!!

File: cg_lims/EPPs/files/file_to_udf.py (hunk 1 above)
Comment (mayabrandi) [reply cg_lims/EPPs/files/file_to_udf.py:49]: Yes. tested. math is not part of pandas.

**Comment by mayabrandi** [comment]
Pandas was removed from requirements in the previous pr already! Merging.

**Pull request merged**